src/main/java/org/apache/maven/plugins/dependency/fromConfiguration/UnpackMojo.java

```diff
@@ -55,7 +55,9 @@ public class UnpackMojo extends AbstractFromConfigurationMojo {
     /**
      * Directory to store flag files after unpack
      */
-    @Parameter(defaultValue = "${project.build.directory}/dependency-maven-plugin-markers")
+    @Parameter(
+            property = "markersDirectory",
```

Comment:
I would like to change it to `mdep.unpack.markersDirectory` as we have for other parameters